utils/build/update_canary_version.js

```diff
@@ -31,6 +31,9 @@ if (process.argv[2] === '--today-date') {
     stdio: ['ignore', 'pipe', 'ignore']
   }).toString('utf8').trim();
   packageJSON.version = `${packageJSON.version}-${timestamp}000`;
+} else if (process.argv[2] === '--commit-sha') {
+  const commitSHA = spawnSync('git', ['rev-parse', 'HEAD'], {cwd: __dirname, encoding: 'utf8'}).stdout.trim();
+  packageJSON.version = `${commitSHA}`;
 } else {
   throw new Error('This script must be run with either --timestamp or --today-date parameter');
```

Comment:
Can we delete the `--commit-timestamp` code path?
```suggestion
  throw new Error('This script must be run with either --commit-sha or --today-date parameter');
```